### ISSUE TITLE
Will esp32S3 be supported later?

### ISSUE THREAD

**Issue by FantasyGmm** [retro-esp32/RetroESP32#103]
ESP32S3 ESP-IDF stable version 4.4 has been released, most features are available, SD card is not supported.
ESP32S3 more powerful than the old ESP32 , built-in PSRAM and FLASH takes up less PCB space

**Comment by 32teeth** [comment]
@FantasyGmm 

Yes, we are looking into the S3 (40+ pins wow), as well as the H2

**Comment by FantasyGmm** [comment]
Very good, I modified the PCB and schematic of ESPlay Micro, changed the main control to ESP32, and the screen to ST7789V, which is an IPS screen, I believe the display effect will be good, I am waiting for the PCB, if there is no problem, I will upload it

**Comment by pebri86** [comment]
> Very good, I modified the PCB and schematic of ESPlay Micro, changed the main control to ESP32, and the screen to ST7789V, which is an IPS screen, I believe the display effect will be good, I am waiting for the PCB, if there is no problem, I will upload it

Wow nice to hear that, don't forget to take advantage of usb feature that esp32s3 has, so the board can be useful to make an usb gadget like directly connected to pc when accessing files or copy rom.

**Comment by FantasyGmm** [comment]
Yes, I use OTG and serial port, as far as I know S3 can support USB JTAG, debugging may be a lot easier

**Comment by FantasyGmm** [comment]
https://oshwlab.com/FantasyGM/esplay_micro_s3,I uploaded the schematic diagram and PCB on oshwlab, I used easyEDA design, so THERE are no other documents, and I am not a professional, so if there is something wrong in the schematic diagram or PCB design, please tell me and how to correct it, thank you

**Comment by FantasyGmm** [comment]
My PCB is still being manufactured, so it has not been tested

**Comment by SSStarku** [comment]
is the retroesp available for esp32s3?

**Comment by 32teeth** [comment]
You'll want to follow https://github.com/retro-esp32/RetroESP32-X project.
We will be updating
